test/rendering/sn-grid-chart/__fixtures__/scenario-1.fix.js

```diff
@@ -0,0 +1,10869 @@
+import gridChart from '../dist/sn-grid-chart';
+
+export default () => ({
+  type: 'sn-grid-chart',
+  load: async () => gridChart,
+  genericObjects: [
+    {
+      getLayout() {
```

Comment:
Why are `senario-1.fix.js` added to Nebula? And if needed, does it really need to have a cube that results in 10k lines?